updateHostsWindows.bat

```diff
@@ -20,11 +20,25 @@ if not exist "%WINDIR%\System32\drivers\etc\hosts.skel" (
 	COPY %WINDIR%\System32\drivers\etc\hosts %WINDIR%\System32\drivers\etc\hosts.skel
 )
 
+:: Make sure we are in the correct working directory
+rem Path info from:
+rem https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call#batch-parameters
+cd %~dp0
+
 :: Update hosts file
-python updateHostsFile.py -a
+if exist updateHostsFile.exe (
+    echo run exe
+    updateHostsFile --auto --extensions fakenews social gambling
+) else (
+    echo run script
+    python updateHostsFile.py --auto --extensions fakenews social gambling

```

Comment:
You can't force default extensions... It's again your scope...

Reply:
The extension was in the `.bat` file when I installed.

Reply:
@evenprimes No please look at the original file present into the master branch!

it was 
```
python updateHostsFile.py -a
```

even your changes describe what was there before ...

Reply:
Ah, sorry, didn't understand the comment!